src/jira/client/jira-client.ts

```diff
@@ -31,6 +32,60 @@ export interface DeploymentsResult {
 	rejectedDeployments?: any[];
 }
 
+
+export interface JiraClient {
+	baseURL: string;
+	issues: {
```

Comment:
Need to move these changes. See https://github.com/atlassian/github-for-jira/pull/2226#discussion_r1255025487

Reply:
I think ill need a hand with this. Looks like `dependabotAlertWebhookHandler` is passed the _old_ jira client. So it looks like a somewhat complicated refactor of the `GithubWebhookMiddleware` that controls what webhooks are called with.